test/suites/devlxd.sh

```diff
@@ -431,3 +431,71 @@ EOF
       lxc project delete "${project}"
   fi
 }
+
+test_snap_devlxd_vm() {
```

Comment:
Can we make this non snap now we have the cached Ubuntu image available to the non snap tests? They don't seem snap specific.

Reply:
Unfortunately for non snap tests, we can only use `--empty` VMs with `migration.stateful=true`. The Ubuntu images for non snap will require the compiling `virtiofsd` which is not ready yet.

Reply:
Oh OK I misunderstood your request. So yeah I am sure they would support it but the bits to cache the `ubuntu-minimal-daily:24.04` image is something I'll implement next week. Until then, would you like me to drop this test?

Reply:
im ok to merge as this allows you to progress with your roadmap item, but id like to revisit in a follow up PR and move any non-snap specific tests into the main test suite run (even if they need to rely on a cached image).